packages/core/src/errors-list.ts

```diff
@@ -0,0 +1,322 @@
+export const ERROR_PREFIX = "IGN";
```

Comment:
Most of this file is basically copied structurally from the similar files in Hardhat.